payments/utils.py

```diff
@@ -176,6 +176,12 @@ def check_payments_status(verbose=0):
             _process_payment(order, payment, verbose)
 
 
+def get_full_order_url(order):
+    ABSOLUTE_URL = settings.get('ABSOLUTE_URL')
+    full_url = '{}order/{}'.format(ABSOLUTE_URL, order.uuid)
```

Comment:
please use [reverse](https://docs.djangoproject.com/en/1.10/ref/urlresolvers/#reverse) this wont work if we change url for some reason...